app/services/RuleProvisionerService.scala

```diff
@@ -79,4 +76,14 @@ class RuleProvisionerService(
   def scheduleUpdateRules(scheduler: Scheduler): Unit = {
     scheduler.scheduleWithFixedDelay(0.seconds, 1.minute)(this)
   }
+
+  private def addLTMatcherToPool(matcherPool: MatcherPool, xmlRules: List[LTRuleXML], defaultRules: List[String] = Nil) = {
+    languageToolFactory.createInstance(xmlRules, defaultRules) match {
+      case Right(matcher) => matcherPool.addMatcher(matcher)
+      case Left(errors) => {
+        logger.error(s"Could not create languageTool instance from ruleResource: ${errors.size} errors found")
+        errors.foreach(e => logger.error(e.getMessage(), e))
```

Comment:
This is going to log the exception message twice, perhaps it could be a static string of "rule resource error" to make it easy to marry up with the line above.